lib/NetPacksBase.h

```diff
@@ -17,13 +17,18 @@ class CArmedInstance;
 class CArtifactSet;
 class CBonusSystemNode;
 struct ArtSlotInfo;
+class CConnection;
 
 #include "ConstTransitivePtr.h"
 #include "GameConstants.h"
 
 struct DLL_LINKAGE CPack
 {
-	CPack() {};
+	// Not serialized. Set in CConnection::retreivePack
+	ui16 type;

```

Comment:
remove this

Reply:
Removed code using it, but totally forgot to remove it from CPack. Thanks.